lib/services/elasticsearch.js

```diff
@@ -473,6 +482,10 @@ class ElasticSearch extends Service {
     assertNoRouting(esRequest);
     assertWellFormedRefresh(this.config.apiVersion, esRequest);
 
+    if (!this.kuzzle.indexCache.exists(esRequest.index, esRequest.type)) {
```

Comment:
why not, but in that case, to be consistent, the check should be done on `update` method too ;)